app/assets/stylesheets/modules/_uploads.styl

```diff
@@ -155,3 +154,10 @@
 
     .upload-viewer-button
       margin 10px
+
+.view_fileDetails
```

Comment:
To follow the conventions of our css class names, this should probably be `view-file-details`, and `row-details`.

These should also not be top-level rules, so that we don't accidentally use the same class name somewhere else. It looks like these two classes would only be used for child elements of `upload-viewer`, so they should be scoped to only apply to child elements, like the rest of the classes in this file.

Reply:
Sorry about this @ragesoss . I'll definitely keep this in mind.